components/Registration/Mitra/index.vue

```diff
@@ -393,6 +416,47 @@ export default {
       } else {
         this.isDropdownOpen = false
       }
+    },
+    fetchUser (token) {
+      try {
+        this.$axios.get('https://www.googleapis.com/oauth2/v2/userinfo', {
+          headers: {
+            Authorization: `Bearer ${token}`,
+            Accept: 'application/json',
+            'Content-Type': 'application/json;charset=UTF-8'
+          }
+        })
+          .then(({ data }) => {
+            this.form.name = data.name
+            this.form.email = data.email
+          })
+      } catch (err) {
+        if (err.responsense?.status === 401) {
```

Comment:
I think this typo should be `err.response` 🤔 ?

Reply:
😆 I think i was typo kak

Reply:
😆 I think i was typo kak

Reply:
😆 I think i was typo kak

Reply:
😆 I think i was typo kak

Reply:
You can see the changes [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/79/commits/a7e7a8c94fee3e482cb5c3256bd8aed3cd2aaa29)